app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>MediaRanker</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title>MediaRanker</title>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+    <%= csrf_meta_tags %>
+  </head>
+  <div class="container">
+    <div class="row">
+      <div class="page-header">
+          <h1><%= link_to "Media Ranker", root_path %> <small>Ranking the Best of Everything</small></h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-lg-12">
+        <body>
```

Comment:
This is confusing. There are several display elements outside the body in this file. Although they may display properly in some modern browsers, I suspect this ordering would wreak havoc on older browsers and might cause issues with interpreting [the DOM](https://www.w3.org/TR/DOM-Level-2-Core/introduction.html) or would prevent your site from displaying as you intend to said users. I suspect that what you're looking for here is `main`, which is a semantic tag used to denote which part of the DOM contains [the main content of a page](http://www.w3schools.com/tags/tag_main.asp).